docs/source/user-guide/configuration/use-condarc.rst

```diff
@@ -49,17 +43,34 @@ The ``.condarc`` file can change many parameters, including:
 
 * Default packages or features to include in new environments.
 
-To create or modify a ``.condarc`` file, use
-the ``conda config`` command or use a text editor to create a
-new file named ``.condarc`` and save it to your user home
-directory or root directory.
+Creating and editing
+====================
+
+The ``.condarc`` file is not included by default, but it is
+automatically created in your home directory the first time you
+run the ``conda config`` command. To create or modify a ``.condarc``
+file, open Anaconda Prompt or a terminal and enter the
+``conda config`` command.
 
 EXAMPLE:
 
 .. code-block:: yaml
 
   conda config --add channels conda-forge
 
+Alternatively, you can open a text editor such as Notepad
+on Windows, TextEdit on MacOS, or VS Code. Name the new file
+``.condarc`` and save it to your user home directory or root
+directory. To edit the ``.condarc`` file, open it from your
+home or root directory and make edits in the same way you would
+with any other text file. If the ``.condarc`` file is in the root
+environment, it will override any in the home directory.
+
```

Comment:
Should we mention here that the .condarc file is interpreted with standard YAML syntax?

Reply:
There is a section above that states it so I pulled that sentence down into the Creating and Editing section.